src/clj/game/cards/hardware.clj

```diff
@@ -938,19 +944,25 @@
               {:runner-trash e :corp-trash e})}
 
    "Qianju PT"
-   {:flags {:runner-phase-12 (req true)}
-    :abilities [{:label "Lose [Click], avoid 1 tag (start of turn)"
-                 :once :per-turn
-                 :req (req (:runner-phase-12 @state))
-                 :effect (effect (update! (assoc card :qianju-active true)))
-                 :msg "lose [Click] and avoid the first tag received until their next turn"}]
-    :events {:corp-turn-ends {:effect (effect (update! (dissoc card :qianju-active)))}
-             :runner-turn-begins {:req (req (:qianju-active card))
-                                  :effect (effect (lose :click 1))}
-             :pre-tag {:req (req (:qianju-active card))
-                       :msg "avoid the first tag received"
-                       :effect (effect (tag-prevent :runner 1)
-                                       (update! (dissoc card :qianju-active)))}}}
+   (let [ability {:label "Lose [Click], avoid 1 tag (start of turn)"
+                  :once :per-turn
+                  :silent (req true)
+                  :req (req (:runner-phase-12 @state))
+                  :effect (effect (update! (assoc card :qianju-active true))
+                                  (lose :click 1))
+                  :msg "lose [Click] and avoid the first tag received until their next turn"}]
+     {:flags {:runner-phase-12 (req true)}
+      :abilities [ability]
+      :events {:corp-turn-ends {:effect (effect (update! (dissoc card :qianju-active)))}
+               :runner-turn-begins
+               {:interactive (req true)
```

Comment:
So you feel this card is important enough to force the Runner to make a decision every turn?

Reply:
There's a trade-off to be made, but I think that a card like this is being played to gain its' effect, so we should ask. If you don't feel that way, then I can easily remove this part.